connect-examples/v2/php_checkout/README.md

```diff
@@ -0,0 +1,28 @@
+Square Checkout Demo
+=========================
+
+This is a simple example application that utilizes Square's Checkout API.
+
+It takes a single payment, declared by the user, and creates an order to use in the Checkout API.
+
+To get it running:
+
+* Clone/download to your local computer.
+* Rename `.envsample` to `.env` and fill in the values.
```

Comment:
I can move "USE_PROD" to ".env" (I meant to do that before, oops), but I use this convention so that the file is not accidentally pushed to a repo with credentials. This would mean renaming and modifying one file instead of two. If we include it in a file that is pushed to GitHub, then if someone else uses this they might accidentally push their credentials and it be viewable publicly. What are your thoughts?

Reply:
Good point about committing credentials. I think using environment variables is a little not obvious that is why, but definitely move them to once place.

Reply:
Given this is an example code, I would keep it simple and direct. The first thing developers want from this sample is to get it run. So, .env shouldn't be a concern if it's not prod code. 

But I agree with you that we should remind developers not submit code with credentials. Can we add a comment such as "WARNING: never upload .env with your credential/access_token", so that they understand they need a .env but not in public location.

Reply:
Ok pushed latest changes. @hukid can you just confirm it's all good before I merge? Thanks!